### ISSUE TITLE
Put diagnostics for "mult_interface_roam" under "Computers" is dashboard  (ros ticket #4985)

### ISSUE THREAD

**Issue by ahendrix** [PR2/pr2_robot#210]
The diagnostics for "multi_interface_roam" are showing up in the "Other" category in the dashboard. If these are a permenant addition to "pr2_bringup/pr2.launch", we should move them into the "Computers" category.

Blaise and Jeremy, are these additions permenant? How long have we been running multi-interface roam on these robots?

trac data:
- Owner: **wim**
- Reporter: **watts**
- Reported at: **Wed Jan 19 10:34:42 2011**
- URL: https://code.ros.org/trac/wg-ros-pkg/ticket/4985

**Comment by ahendrix** [comment]
[leibs] Multi interface roam is the solution for the lucid robot networking.  It is not present on the jaunty robots.

**Comment by ahendrix** [comment]
[blaise] The way these diagnostics are done will probably change in version 2 of multi_interface_roam. Version 2 is the version that would most likely work well with ralink adapters. Is it worth addressing this ticket now or once V2 is ready?

**Comment by ahendrix** [comment]
[watts] Wim, are you looking at this?

**Comment by ahendrix** [comment]
[wim] I released a patch to the analyzers last week, but then I noticed yesterday that the multi_interface_roam diagnostics are not always getting published.  Is the multi_interface_roam part of robot.launch, or part of the system startup?

**Comment by ahendrix** [comment]
[leibs] It's part of system startup.  When we started looking at this in the context of unstable/diamondback, we realized it didn't actually make sense to run multi_interface_roam in the context of the main robot master.  It currently runs against a "system master" (which may or may not be running) on port 10301.

I'm inclined to say we shouldn't bother with the analyzers for it.  In the distant future once multi-master stuff has been properly solved we may want to figure out what the right way is to map it from the system master into the ros master.  But doing so exposes the risks associated with trying to merge cturtle and unstable nodes into the same master (which is why we split them in the first place).
